generate/src/contract.rs

```diff
@@ -138,7 +138,7 @@ impl Default for Context {
 
 pub(crate) fn expand(args: Args) -> Result<TokenStream> {
     let cx = Context::from_args(args)?;
-    let contract = expand_contract(&cx).context("error expanding contract from source")?;
+    let contract = expand_contract(&cx).context("error expanding contract from its ABI")?;
```

Comment:
This change appears to only be replacing the word "source" with "its ABI" in an an error string and does not appear to have anything to do with the PR's description.

Reply:
Hmm.. It appears to be because I merged things in the wrong order. This was the original change: